plugins/arrow-flight-rpc/src/test/java/org/opensearch/arrow/flight/bootstrap/tls/SslContextProviderTests.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.arrow.flight.bootstrap.tls;
+
+import org.opensearch.plugins.SecureTransportSettingsProvider;
+import org.opensearch.test.OpenSearchTestCase;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class SslContextProviderTests extends OpenSearchTestCase {
```

Comment:
To be fair, this tests does not really bring much value (we are testing mockito, literally), I would remove it